scripts/core/program.yml

```diff
@@ -182,6 +182,39 @@ returns:
       - "If an error occurred when building `hProgram`."
 --- #--------------------------------------------------------------------------
 type: function
+desc: "Produces an executable program from one program, negates need for the linking step."
+class: $xProgram
+name: BuildExp
```

Comment:
This should be defined in a separate file, since its experimental, e.g. `scripts/core/exp-multi-device-compile.yml`.

Reply:
Fixed

Reply:
I've addressed this in the commit I added to this PR.